backend/api/exceptions.py

```diff
@@ -12,71 +12,44 @@ class ApiKeyCreateException(APIException):
     status_code = 500
     default_detail = "Exception while create API key"
 
-    def __init__(self, detail: Optional[str] = None, code: Optional[int] = None):
-        if detail is not None:
-            self.detail = detail
-        if code is not None:
-            self.code = code
-        super().__init__(detail, code)
-
```

Comment:
@chandrasekharan-zipstack why have we removed this init? Is it because the parent class have same implementation?

Reply:
Yes, this handling is redundant since DRF's [APIException](https://github.com/encode/django-rest-framework/blob/master/rest_framework/exceptions.py#L99) does it